frontend/src/pages/spells/[slug]/index.js

```diff
@@ -0,0 +1,50 @@
+import SpellDetails from '../../../components/pages/spells/[slug]/SpellDetails'
+import { getSpells, getSpellsBySlug } from '../../../lib/load_spells';
+
+
+const Spells = ({ data, links }) => {
+  const { attributes } = data;
+
+  return (
+    <SpellDetails attributes={attributes} apiLink={links?.self} />
+  )
+}
+
+export async function getStaticPaths() {
+  const fetchedSpells = await getSpells({ perPage: 30 });
+
+  const spellSlugs = fetchedSpells?.data?.map(
+    (spell) => spell?.attributes?.slug
+  );
```

Comment:
It would be a good addition if we pre-load some famous spells such as Expecto Patronum, Lingardium Leviosa, Stupefy, Imperio, Avada Kedavra etc. You can checkout how I did it in the [single character page](https://github.com/danielschuster-muc/potter-db/blob/master/frontend/src/pages/characters/%5Bslug%5D/index.js#L21-L42) as a reference. Probably 7-10 spells will sufficient.